cmd/bots/discord/discord.go

```diff
@@ -45,14 +45,15 @@ type discordBotConfig struct {
 	logLevel        string
 	development     bool
 	bindAddress     string
+	scheme          string
 }
 
 func newDiscordBotConfigConfig() *discordBotConfig {
 	c := new(discordBotConfig)
-	tools.StringVarFlagWithEnv(&c.natsPubSubURL, "nano-msg-pubsub-url",
-		"ipc:///tmp/discord/nano-msg-nodemon-pubsub.ipc", "Nanomsg IPC URL for pubsub socket")
-	tools.StringVarFlagWithEnv(&c.natsPairURL, "nano-msg-pair-discord-url",
-		"ipc:///tmp/nano-msg-nodemon-pair.ipc", "Nanomsg IPC URL for pair socket")
+	tools.StringVarFlagWithEnv(&c.natsPubSubURL, "nats-pubsub-url",
+		"nats://127.0.0.1:4222", "NATS server URL for pubsub messaging")
+	tools.StringVarFlagWithEnv(&c.natsPairURL, "nats-pair-discord-url",
```

Comment:
Is it still necessary for discord?

Reply:
You can still type some commands as far as I remember